docs/features/details/Details.md

```diff
@@ -2,18 +2,38 @@
 
 Details are a generic data structure used for storing additional (application specific) information inside the Scenarioo documentation.
 
-This `details` are simple maps of key-value-pairs. Each `entry` in such a details map has a `key` and a `value`. 
+This `details` are simple maps of key-value-pairs that define further properties of an object. Each such  property which has a `key` and a `value`.
 
-The `key` is always a simple string (the name of the information). 
+## Details - Class Diagram
 
-The `value` of such an entry can be of different kinds:
+
+This class diagram shows the generic data structure of what you can store in a details map as values:
+
+```puml { src="features/details/class-diagram.puml" }
+```
+
+## Details - Example Object Diagram
+
+Here is such an example of such a data structure you could add like this to the details of a scenario object:
```

Comment:
"Here is an example of such a data structure that you could add to the details of a scenario object:"

Reply:
Yes, I have to improve my writting! Damn good improvement! Great! :-D 👍